pkg/network/networkutils/netlink.go

```diff
@@ -0,0 +1,167 @@
+// +build linux
```

Comment:
we need a no-op replacement for other platforms

Reply:
for each method?

Would it be okay to add a replacement when composing the final app ?

For example, the final app could have two files `app.go and app_others.go`.
Then `app_others.go` could simply print that running it is not supported on that platform.

Reply:
yes, could work too

Reply:
> Would it be okay to add a replacement when composing the final app ?

Seems like the library shouldn't just flat out fail to compile on other platforms.